conans/client/remover.py

```diff
@@ -47,7 +48,7 @@ def remove(self, pattern, src=None, build_ids=None, package_ids_filter=None, for
                     if not src and build_ids is None and package_ids_filter is None:
                         remover.remove(conan_ref)
```

Comment:
Should not remove the registry entry despite of being a local remove? (It's not removed, I've tried)